atomix/cluster/src/test/java/io/atomix/cluster/messaging/impl/NettyMessagingServiceTest.java

```diff
@@ -429,4 +432,214 @@ public void shouldNotBindToAdvertisedAddress() {
     assertThat(messagingService.bindingAddresses()).contains(bindingAddress);
     assertThat(messagingService.address()).isEqualTo(nonBindableAddress);
   }
+
+  @Test
+  public void testRemoteHandlerFailure() {
+    // given
+    final String exceptionMessage = "foo bar";
+    final RemoteHandlerFailure execptedException =
```

Comment:
:wrench: Feel free to use `var` keyword when the type is obvious from the assignment. For example, here it's pretty clear, but lower when declaring a `BiFunction` it's not, so it makes sense to keep it. However, we don't have a strong guideline on this and it's more of a suggestion.

Reply:
Thanks! I applied the requested changes.